### PR TITLE
Ensure that queues are not declared in config file and allow overrides on CLI

### PR DESCRIPTION
**Pull request by fzf**

### CODE REVIEW

File: spec/lib/simplekiq/config_spec.rb [hunk 1]
```diff
@@ -11,7 +13,7 @@
         Sidekiq.options[:lifecycle_events][:startup].first.call
       }.to change{
         Sidekiq.options[:queues]
-      }.from([]).to(['queue'])
+      }.from(['default']).to(['queue'])
```
Comment (sidt101) [inline spec/lib/simplekiq/config_spec.rb]: Seeing this spec fail locally:

```
 1) Simplekiq::Config#config sets Sidekiq options
     Failure/Error: Sidekiq.options[:lifecycle_events][:startup].first.call

     NoMethodError:
       undefined method `call' for nil:NilClass
     # ./spec/lib/simplekiq/config_spec.rb:13:in `block (4 levels) in <top (required)>'
     # ./spec/lib/simplekiq/config_spec.rb:12:in `block (3 levels) in <top (required)>'

Finished in 0.03035 seconds (files took 0.26922 seconds to load)
14 examples, 1 failure

Failed examples:

rspec ./spec/lib/simplekiq/config_spec.rb:11 # Simplekiq::Config#config sets Sidekiq options
```

**Comment by hex337** [comment]
So, this is going to prevent people from setting up queues in yaml if they use the gem? That's probably fine, just want to make sure I understand what this is doing.

**Comment by fzf** [comment]
@hex337 yes you are correct, they will be able to set queues using the CLI but not the yaml file since the workers should know what queue they are in

File: spec/lib/simplekiq/config_spec.rb (hunk 1 above)
Comment (fzf) [reply spec/lib/simplekiq/config_spec.rb]: @sidt101 it passes CI ¯\_(ツ)_/¯

Comment (sidt101) [reply spec/lib/simplekiq/config_spec.rb]: 🤷‍♂️

**Pull request merged**